app/specific/ScreensObj.js

```diff
@@ -1235,11 +1235,12 @@ function ScreensObj_InitSearchChannels() {
         table: 'stream_table_search_channel',
         screen: Main_SearchChannels,
         object: 'channels',
-        base_url: Main_kraken_api + 'search/channels?limit=' + Main_ItemsLimitMax + '&query=',
+        use_helix: true,
+        base_url: Main_helix_api + 'search/channels?first=' + Main_ItemsLimitMax + '&query=',
```

Comment:
```suggestion
        base_url: `${Main_helix_api}search/channels?first=${Main_ItemsLimitMax}&query=`,
```

Reply:
This gives error `'template literal syntax' is only available in ES6 (use 'esversion: 6')`. I ignore this for the moment.